cves/CVE-2017-11665.yml

```diff
@@ -281,8 +280,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: true
+  note: "Vulnerability relates to remote video sources"
```

Comment:
But does it have to do with the part of the system that interacts with IPC? Just because it's input doesn't mean it's IPC.